packages/data/tests/unit/adapters/dimensions/elide-test.ts

```diff
@@ -4,36 +4,69 @@ import ElideDimensionAdapter from 'navi-data/adapters/dimensions/elide';
 import { DimensionColumn } from 'navi-data/adapters/dimensions/interface';
 import { RequestV2, AsyncQueryResponse, QueryStatus, RequestOptions } from 'navi-data/adapters/facts/interface';
 import DimensionMetadataModel from 'navi-data/models/metadata/dimension';
+import NaviMetadataService from 'navi-data/services/navi-metadata';
+import { TestContext as Context } from 'ember-test-helpers';
+import ElideOneScenario from 'dummy/mirage/scenarios/elide-one';
+import ElideTwoScenario from 'dummy/mirage/scenarios/elide-two';
+// @ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { Server } from 'miragejs';
+
+interface TestContext extends Context {
+  metadataService: NaviMetadataService;
+  server: Server;
+}
 
 const fakeResponse: AsyncQueryResponse = {
   asyncQuery: { edges: [{ node: { id: '1', query: 'foo', status: QueryStatus.COMPLETE, result: null } }] }
 };
 
 module('Unit | Adapter | Dimensions | Elide', function(hooks) {
   setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function(this: TestContext) {
+    this.metadataService = this.owner.lookup('service:navi-metadata');
+    ElideOneScenario(this.server);
+    await this.metadataService.loadMetadata({ dataSourceName: 'elideOne' });
```

Comment:
Are we using `elideOne` in this test file?

Reply:
Yeah we load a dimension metadata model from the metadata service